include/swift/AST/DiagnosticsSema.def

```diff
@@ -6345,6 +6345,17 @@ ERROR(invalid_objc_swift_rooted_class,none,
 NOTE(invalid_objc_swift_root_class_insert_nsobject,none,
      "inherit from 'NSObject' to silence this error", ())
 
+WARNING(objc_header_sorting_arbitrary,none,
```

Comment:
Is it possible to add a test for this diagnostic or we have no known cases that would trigger this?

Reply:
We have no known cases that would trigger this, so I'm instead testing that it isn't emitted by turning on `-verify` everywhere I can. 

(If you think it's really important, I could add a flag that deliberately breaks the sort for testing purposes.)